Crypt/Mechanisms/Check.swift

```diff
@@ -213,11 +216,6 @@ class Check: CryptMechanism {
   }
 
   fileprivate func getSkipUsers() -> Bool {
-    let uid : uid_t = self.uid
-    NSLog("%u", uid)
-    if (uid < 501) {
-      return true
-    }
     guard let prefValue = CFPreferencesCopyAppValue("SkipUsers" as CFString, bundleid as CFString) as? [String]
       else { return false }
     guard let username = self.username
```

Comment:
nit: move L221-222 above L219.
If we can't unwrap the username, there is no sense in doing a pref lookup.